crates/red_knot_python_semantic/resources/mdtest/shadowing/function.md

```diff
@@ -25,3 +25,27 @@ def f(): ...
 
 f: int = 1
 ```
+
+## Explicit shadowing involving `def` statements
+
```

Comment:
```suggestion
## Explicit shadowing involving `def` statements

Since a `def` statement is a declaration, one `def` can shadow another `def`, or shadow a previous non-`def` declaration, without error.

```